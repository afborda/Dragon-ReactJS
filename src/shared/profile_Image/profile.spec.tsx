import { render } from '@testing-library/react'
import React from 'react'
import ProfileImage from '.'

describe('<ProfileImage>', () => {
  it('should  render the heading', () => {
    const { container } = render(<ProfileImage />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
