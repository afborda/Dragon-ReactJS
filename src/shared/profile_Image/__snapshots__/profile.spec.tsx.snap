// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileImage> should  render the heading 1`] = `
<div>
  <img
    alt="Image de perfil do usuario"
    class="style__CustomImg-gVgzYn iFHTNp"
    src="test-file-stub"
  />
</div>
`
